openedx/core/djangoapps/oauth_dispatch/migrations/0001_initial.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.OAUTH2_PROVIDER_APPLICATION_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
```

Comment:
FYI @edx/devops on this upcoming migration.  It shouldn't be a problem - since it's a new table.  However, keep in mind that this table has a foreign relationship to the OAuth2 `Application` table.